apps/nextjs-website/react-components/components/BannerLink/BannerLink.tsx

```diff
@@ -56,14 +56,17 @@ const BannerLink = ({ theme, sections, sectionID }: BannerLinkProps) => {
                   theme === 'light'
                     ? lightBackgrounds[index % 2]
                     : darkBackgrounds[index % 2],
-                padding: { md: '64px 24px', xs: '32px 24px' },
                 width: '100%',
                 flex: 1,
                 display: 'flex',
                 justifyContent: 'center',
               }}
             >
-              <Stack textAlign='center' alignItems='center'>
+              <Stack
+                textAlign='center'
+                alignItems='center'
+                sx={{ margin: { md: '64px 24px', xs: '32px 24px' } }}
```

Comment:
```suggestion
                margin: { md: '64px 24px', xs: '32px 24px' }
```
You should be able to just set margin without going through sx.